src/utils/structuredData.ts

```diff
@@ -3,9 +3,9 @@ import { secondsToISO8601 } from './datetime';
 
 import type { Playlist, PlaylistItem } from '#types/playlist';
 
-export const generateSeriesMetadata = (seriesPlaylist: Playlist) => {
-  // @todo this still used the series route
-  const seriesCanonical = `${window.location.origin}/s/${seriesPlaylist.feedid}`;
+export const generateSeriesMetadata = (seriesPlaylist: Playlist, mediaId: string | undefined) => {
+  // Use playlist for old flow and media id for a new flow
+  const seriesCanonical = `${window.location.origin}/m/${seriesPlaylist.feedid || mediaId}`;
```

Comment:
I think it's valuable to keep the `/s` routing for series.

Reply:
I have answered in the thread regarding `Series` component / page.